.travis.yml

```diff
@@ -39,26 +38,26 @@ jobs:
         - pip install --user --upgrade pip
         - pip install --user yamllint
         # install golangci-lint
+        # workaround for https://github.com/golangci/golangci-lint/issues/575
         - curl -sf
-          "https://install.goreleaser.com/github.com/golangci/golangci-lint.sh"
+          "https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh"
           | bash -s -- -b $GOPATH/bin "${GOLANGCI_VERSION}"
       script:
         - scripts/lint-text.sh --require-all
         - scripts/lint-go.sh
         - scripts/test-go.sh
 
-    - name: cephcsi
+    - name: cephcsi with kube 1.13.7
       script:
         - scripts/skip-doc-change.sh || travis_terminate 0;
```

Comment:
We can possibly move this line up to the `static-check-make` section (last line) and hence skip the check in future sections like in line 58 in the new file.

Can be taken care the next time we edit this file.

Reply:
yes nice idea will update if it works